### PR TITLE
Update doc nightly [skip ci]

### PR DESCRIPTION
**Pull request by YanxuanLiu**
1. Modified `docs.sh` to make it get Github account and token from ENV.
2. Modified `test.sh` to make it suitable for both nightly and release pipeline.

### CODE REVIEW

File: ci/docs.sh [hunk 1]
```diff
@@ -51,9 +51,12 @@ cp -r ../build/html/* $api_dest/
 
 git add --all
 dff=$(git diff --staged --stat)
+repo_url=$(git config --get remote.origin.url)
+url=${repo_url#https://}
+github_account=${GITHUB_ACCOUNT:-nvauto}
 if [[ -n $dff ]]; then
     git commit -m "${TAG}"
-    git push origin gh-pages
+    git push -f https://${github_account}:${GITHUB_TOKEN}@${url} gh-pages
```
Comment (eordentlich) [inline ci/docs.sh:59]: Curious where GITHUB_TOKEN is being defined.

**Review by eordentlich — DISMISSED** [review]
👍

File: ci/docs.sh (hunk 1 above)
Comment (pxLi) [reply ci/docs.sh:59]: it will be defined in the internal nightly/release pipelines

File: ci/test.sh [hunk 2]
```diff
@@ -54,4 +54,12 @@ if [[ $type == "nightly" ]]; then
     # need to invoke docs.sh from top level of repo
     cd .. # top level of repo
     ci/docs.sh nightly
+elif [[ $type == "release" ]]; then
```
Comment (pxLi) [inline ci/test.sh:57]: we do not push release doc while release build, can you remove this part?

**Comment by pxLi** [comment]
build

**Review by pxLi — APPROVED** [review]

**Comment by pxLi** [comment]
build

**Comment by YanxuanLiu** [comment]
build

**Pull request merged**